src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/InteractiveRequest.cs

```diff
@@ -65,6 +63,7 @@ internal class InteractiveRequest : RequestBase
             cancellationToken.ThrowIfCancellationRequested();
             MsalTokenResponse tokenResponse = await GetTokenResponseAsync(cancellationToken)
                 .ConfigureAwait(false);
+            Metrics.TotalAccessTokensFromIdP++;
```

Comment:
Since you access a shared variable from multiple threads, the 2 operations (add and assign) might not happen at once. You can use smth like Interlocked.Increment to have "add" as an atomic operation. See https://docs.microsoft.com/en-us/dotnet/api/system.threading.interlocked.increment?view=net-5.0